support/k8s/k8s-workload-registrar/mode-crd/api/spiffeid/v1beta1/spiffeid_webhook.go

```diff
@@ -102,9 +103,17 @@ func (s *SpiffeID) validateSpiffeID() error{
 		return errs.New("spec.spiffeId must begin with " + spiffeIDPrefix)
 	}
 
-	// Ensure namespace selector matches namespace of Spiffe ID resource
-	if s.Spec.Selector.Cluster == "" && s.ObjectMeta.Namespace != s.Spec.Selector.Namespace {
-		return errs.New("spec.Selector.Namespace must match namespace of resource")
+	if (s.Spec.Selector.Cluster != "" || s.Spec.Selector.AgentNodeUid != "") {
```

Comment:
nit: `make lint` would complain about this if it wasn't excluded. The parens are unnecessary. Should this webhook impl life somewhere that is linted since it isn't purely generated code?

Reply:
Thanks, good catch. I updated the .golangci-lint to only exclude spiffe_types.go which is the only file the linter complains about. The parens have been removed.